packages/material-ui/src/CircularProgress/CircularProgress.test.js

```diff
@@ -93,6 +102,14 @@ describe('<CircularProgress />', () => {
       );
       expect(circularProgress).to.have.attribute('aria-valuenow', '70');
     });
+
+    it('issues a warning for variant="static"', () => {
+      render(<CircularProgress variant="static" value={70} />);
```

Comment:
For actual `.propTypes` warnings prefer `PropTypes.checkProptypes` (the codebase has some example usage). And don't forget to `PropTypes.resetWarningCache` in `beforeEach`. Otherwise the test will cause problems in watchmode (and with React 17)

Reply:
> For actual .propTypes warnings prefer PropTypes.checkProptypes

Oh, okay. Previously you'd said "minus the checkPropTypes part": https://github.com/mui-org/material-ui/pull/22094#discussion_r467040508

Reply:
Nah, I misspoke. I thought you weren't using `.propTypes` where `render` is fine.